c2corg_ui/static/partials/cards/outings.html

```diff
@@ -132,7 +132,13 @@
             </div>
           </a>
         </div>
-      </div>    
+      </div>
```

Comment:
this is not related to your PR, but there is a typo on line 80 (double quotes). Could you also correct this?

Reply:
Also, on line 128, the tag is not closed, the greater char is missing.

Reply:
fixed